app/controllers/concerns/api/participant_actions.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Api
+  module ParticipantActions
+    extend ActiveSupport::Concern
+
+    def withdraw
+      perform_action(service_namespace: ::Participants::Withdraw)
+    end
+
+  private
+
+    def perform_action(service_namespace:)
+      params = HashWithIndifferentAccess.new({ cpd_lead_provider: current_user, participant_id: participant_id }).merge(permitted_params["attributes"] || {})
+      profile = recorder(service_namespace: service_namespace, params: params).call(params: params)
+      render json: ParticipantSerializer.new(profile.user).serializable_hash.to_json
```

Comment:
Are you going to make this a local private method so this can just be shared between the two controllers?